torch/csrc/distributed/c10d/ProcessGroup.hpp

```diff
@@ -48,6 +48,77 @@ namespace c10d {
 // process group to find each other (referred to as rendezvous from
 // hereon)
 //
+  // We share dimensionality about the sparse tensors before collecting
+  // their contents. We assume here that the maximum number of sparse
+  // and dense dimensions is 4. This is stored in a contiguous piece of
```

Comment:
nit (before this PR):
```suggestion
  // and dense dimensions is 4 each. This is stored in a contiguous piece of
```